pyproject.toml

```diff
@@ -10,7 +10,7 @@ python = "^3.8,<3.11"
 defusedxml = "^0.7.1"
 
 [tool.poetry.group.dev.dependencies]
-demisto_sdk = "1.14.0"
+demisto-sdk = {git = "https://github.com/demisto/demisto-sdk.git", rev="789278337995c786e55b80f50c58d6410f70689b"}
```

Comment:
@dansterenson please confirm this action with @dorschw

Reply:
do not merge. let's discuss it offline

Reply:
It was just for testing. As written, it should be merged after [this](https://github.com/demisto/demisto-sdk/pull/2722) is merged.

Reply:
This file will be reverted once the SDK version is released.